### PR TITLE
[DOCS] Update Get Started link

### PR DESCRIPTION
**Pull request by Stuzanna**
Getting started link was a 404, I believe this is the best substitute.

https://github.com/great-expectations/great_expectations/issues/10321

There was also [Introduction to GX Core](https://docs.greatexpectations.io/docs/core/introduction/) but I thought this picked up the journey in a better location.
The link on contributing in this PR template is also out of date.

- [x] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`) - *NA*
- [x] Appropriate tests and docs have been updated - *N/A*

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

**Comment by Stuzanna** [comment]
@cla-bot check

**Comment by joshzzheng** [comment]
Hey @Stuzanna Thanks for the PR! The link should point to our Try GX doc here: https://docs.greatexpectations.io/docs/core/introduction/try_gx

If you can get that updated, we'll get this merged.

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ For data sources and other integrations that GX supports, see [GX integration su
 
 ## Get started
 
-GX recommends deploying GX Core within a virtual environment. For more information about getting started with GX Core, see [Get started with Great Expectations](https://docs.greatexpectations.io/docs/oss/tutorials/quickstart).
+GX recommends deploying GX Core within a virtual environment. For more information about getting started with GX Core, see [Get started with Great Expectations](https://docs.greatexpectations.io/docs/core/introduction/try_gx).
```
Comment (Stuzanna) [inline README.md]:
```suggestion
GX recommends deploying GX Core within a virtual environment. For more information about getting started with GX Core, see [Get started with Great Expectations](https://docs.greatexpectations.io/docs/core/introduction/try_gx/).
```

**Comment by Stuzanna** [comment]
Hiya, I think it does, added trailing slash if that's what you meant?

**Comment by joshzzheng** [comment]
@Stuzanna yup you're right - I thought I saw another comment pointing to the landing page instead of the try GX page. We'll get this merged soon. Thanks!

**Review by klavavej — COMMENTED** [review]
Thank you so much for this contribution and apologies for our delay in reviewing it. This issue has already been resolved in the time since you opened this PR, so I will be closing this PR. Thanks again!

**Pull request closed without merging**